docs/api/web-contents.md

```diff
@@ -874,20 +874,47 @@ Returns:
 * `event` Event
 * `dirtyRect` [Rectangle](structures/rectangle.md)
 * `image` [NativeImage](native-image.md) - The image data of the whole frame.
+* `texture` [OffscreenSharedTexture](structures/offscreen-shared-texture.md) - The GPU shared texture of the frame. Will be `null` if `webPreferences.offscreenUseSharedTexture` is `false`.
```

Comment:
Instead of extending the argument list, let's put stuff on the event. See https://github.com/electron/governance/blob/main/wg-api/best-practices.md#how-will-this-api-be-extended-in-the-future and https://github.com/electron/electron/pull/37085/files#diff-261075c0abf5bfd33be70313bcd3699da9b1222b6f27df0e8294fc63789f160bR1795-R1807 for an example. The C++ side of this is still a little rough but it beats long inscrutable argument lists.

Reply:
SGTM!